doc/mgr/rgw.rst

```diff
@@ -50,10 +50,10 @@ Following is an example of RGW multisite spec file:
     rgw_frontend_port: 5500
 
 .. note:: The spec file used by RGW has the same format as the one used by the orchestrator. Thus,
-          the user can provide any orchestration supported rgw parameters including advanced
+          the user can provide any orchestration supported RGW parameters including advanced
```

Comment:
RGW multisite spec files follow the same format at cephadm orchestrator spec files.  Thus the user can specify any supported RGW parameters